lms/djangoapps/courseware/date_summary.py

```diff
@@ -204,10 +204,20 @@ class CertificateAvailableDate(DateSummary):
     title = ugettext_lazy('Certificate Available')
 
     @property
+    def active_certificates(self):
+        return [
+            certificate for certificate in self.course.certificates.get('certificates', [])
```

Comment:
I'm not sure if there is an edX style for this but this looks fine and readable to me.

Reply:
:1st_place_medal: